modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -54,4 +54,14 @@ for the route.
 |`replicas`
 |Replica count for the registry.
 
+|`spec.storage.managementState`
+
+|The Image Registry Operator sets the `spec.storage.managementState` parameter to `Managed` on new installations or upgrades of installer-provisioned infrastructure clusters on AWS or Azure.
+
+* `Managed`: Determines that the Image Registry Operator manages underlying storage. If the Image Registry Operator's `managementState` is set to `Removed`, then the storage is deleted.
+** If the `managementState` is set to `Managed` the Image Registry Operator attempts to apply some default configuration on the underlying storage unit. For example, if set to `Managed` the Operator tries to enable encryption on the S3 bucket before making it available to the registry. If you do not want the default settings to be applied on the storage you are providing make sure the `managementState` is set to `Unmanaged`.
+* `Unmanaged`: Determines that the Image Registry Operator leaves the storage settings alone. If the Image Registry Operator's `managementState` is set to `Removed`, then the storage is not deleted. If you provided an underlying storage unit configuration, such as a bucket or container name, and the `spec.storage.managementState` is not yet set to any value, then the Image Registry Operator configures it to `Unmanaged`.
```

Comment:
Maybe `Determines that the Image Registry Operator ignores the storage settings.` ?